python/cudf/cudf/core/column/numerical.py

```diff
@@ -211,12 +212,30 @@ def _binaryop(self, other: ColumnBinaryOperand, op: str) -> ColumnBase:
             np.uint64: np.float64,
             np.bool_: np.float32,
         }
+        if cudf.get_option("mode.pandas_compatible"):
+            int_float_dtype_mapping = {
```

Comment:
Maybe vanilla cuDF should just always return float64 just to match pandas? cc @vyasr